js/test/Ice/proxy/Client.ts

```diff
@@ -416,6 +416,39 @@ export class Client extends TestHelper {
         id2 = Ice.stringToIdentity(idStr);
         test(id.equals(id2));
 
+        //
+        // Test proxies with duplicated endpoint option.
+        //
+
+        let objPrx = communicator.stringToProxy("hello:tcp -h host1 -p 4061");
```

Comment:
We should use `defaultProtocol` defined at the begining, for browser compatibility. I think you cannot parse TCP endpoints in a browser can you?

Reply:
We're running this test in a browser?